docs/src/prototypes/customToolbar/darkThemeOverrides.ts

```diff
@@ -0,0 +1,273 @@
+
+            background: v.ctItemBackground,
+            borderStyle: v.ctBorderStyle,
+            borderWidth: v.ctBorderWidth,
+            borderRadius: 0,
+            height: v.ctHeight,
+            minWidth: v.ctHeight,
+            color: v.ctItemColor,
+
+            ...(p.active &&
+              !v.isCtItemPrimary && {
+                // active intentionally before primary and danger, only affects regular items
+                color: v.ctItemActiveColor,
+                background: v.ctItemActiveBackground,
+
+                '::before': {
+                  content: `''`,
+                  position: 'absolute',
+                  top: `-${v.ctBorderWidth}`,
+                  left: `-${v.ctBorderWidth}`,
+                  bottom: `-${v.ctBorderWidth}`,
+                  right: `-${v.ctBorderWidth}`,
+                  background: v.ctItemActiveBackgroundOverlay,
+
+                  ...(p.isFromKeyboard && {
+                    borderStyle: v.ctBorderStyle,
+                    borderWidth: v.ctBorderWidth,
+                    borderColor: v.ctItemBorderColorFocus,
+                  }),
+                },
+              }),
+
+            ...(v.isCtItemDanger && {
+              background: v.ctItemDangerBackground,
+            }),
+
+            ...(v.isCtItemPrimary && {
+              background: v.ctItemPrimaryBackground,
+            }),
+
+            ':hover': {
+              color: v.ctItemColorHover,
+              background: v.ctItemBackgroundHover,
+
+              ...(v.isCtItemDanger && {
+                color: v.ctItemDangerColorHover,
+                background: v.ctItemDangerBackgroundHover,
+              }),
+
+              ...(v.isCtItemPrimary && {
+                color: v.ctItemPrimaryColorHover,
+                background: v.ctItemPrimaryBackgroundHover,
+              }),
+            },
+
+            ...(v.isCtItemWithNotification && {
+              '::after': {
+                content: `''`,
+                position: 'absolute',
+                width: v.ctItemNotificationSize,
+                height: v.ctItemNotificationSize,
+                borderRadius: '50%',
+                background: v.ctItemNotificationBackgroundColor,
+                transform: 'translateX(100%) translateY(-100%)',
+              },
+            }),
+
+            ...(p.isFromKeyboard && {
+              background: v.ctItemBackgroundHover,
+              borderColor: v.ctItemBorderColorFocus,
+              color: v.ctItemColorFocus,
+
+              ...(v.isCtItemDanger && {
+                color: v.ctItemDangerColorHover,
+                background: v.ctItemDangerBackgroundHover,
+              }),
+
+              ...(v.isCtItemPrimary && {
+                color: v.ctItemPrimaryColorHover,
+                background: v.ctItemPrimaryBackgroundHover,
+              }),
+            }),
+          }),
+
+          ...(v.isCtItemIconNoFill && {
+            '& .ui-icon__filled': {
```

Comment:
Can we use here `getIconFilledOrOutlineStyles.ts`? Or at least can we use `[`& .${teamsIconClassNames.filled}`]` and `[`& .${teamsIconClassNames.outline}`]` as selectors?

Reply:
Unfortunately there is currently no way how to access it from the outside of the library.
This is a bummer we should think about.